resources/metadata/kubernetesDeploy.yaml

```diff
@@ -223,6 +223,24 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: valuesMapping
+        type: "map[string]interface{}"
+        longDescription: |
+          Mapping of values provided by Piper onto custom paths in format `[custom-path]: [piper-value]`
+
+          Example:
+          ```yaml
+          valuesMapping:
+            subchart1.image.tag:        image.tag
+            subchart2.image.tag:        image.debug.tag
+            subchart2.image.repository: image.debug.repository
+            subchart2.image.pullsecret: secret.dockerconfigjson
+            mysidecar:                  image.sidecar.gun
```

Comment:
I guess the full `gun` is not yet available. This could be added as well, or we need to change the example.